libraries/GCS_MAVLink/GCS_Param.cpp

```diff
@@ -53,7 +53,9 @@ GCS_MAVLINK::queued_param_send()
 
     // use at most 30% of bandwidth on parameters. The constant 26 is
     // 1/(1000 * 1/8 * 0.001 * 0.3)
```

Comment:
It's not a new issue, but can anyone explain the comment here? I'm glad we decided to document where the magic 26 came from, but it's documented with a bunch of other magic numbers, which doesn't seem to improve the situation much.

Reply:
Well...
One constant for converting kilobits to bits.
One constant for converting bits to bytes.
One constant for converting milliseconds to seconds.
One constant for the 30%.

I think I might have those right.  I'm not sure I know which is which :-)

Reply:
I wasn't sold originally since it's normally 8N1 that's actually `57600*8/10 = 46080` bits of actual data (in the old version), so it seemed like something was still missing.